crypto/bindings/field.ts

```diff
@@ -95,12 +95,13 @@ function createFieldBindings(Field: FiniteField) {
     random(): Field {
       return [0, Field.random()];
     },
+    // TODO
     rng(i: number): Field {
       console.warn('rng is not implemented');
       return [0, Field.random()];
     },
-    to_bigint(x: Field): Bigint256 {
```

Comment:
?

Reply:
Ok, I see. Can you add a comment in the code explaining this please? For our future us :).

Reply:
Don't we want a deep copy? I.e. copying x deeply too?

Reply:
x is a `bigint` which is a JS primitive value. it can't be mutated, so no need (and no way) to deep copy